src/model/config.hpp

```diff
@@ -277,15 +277,15 @@ class Config
         , use_ansi_color(true)
         , use_backspace(true)        
         , hidden_input_character('*')        
-        , use_auto_validate_users(false)
+        , use_auto_validate_users(true)
         , use_newuser_voting(false)
         , use_auto_kick_unvalidated(false)        
         , newuser_votes_validate(3)
         , newuser_votes_delete(3)
         , newuser_days_to_upload(7)
         , days_keep_logs(30)                
         , qwk_packet_name("Obv2_XRM")
-        , starting_menu_name("top")
+        , starting_menu_name("main")
```

Comment:
Default Starting menu is Main right now until more flags and Post Login Sequence utilities are coded.